<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDownLoader</class>
 <widget class="QMainWindow" name="QtDownLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDownLoader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>461</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lab_url">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="ledit_url"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lab_path">
         <property name="text">
          <string>路径</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledit_path"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lab_threadnum">
         <property name="text">
          <string>线程数量</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>24</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lab_prog">
         <property name="text">
          <string>进度</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QProgressBar" name="prog_download">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="lab_prog_num">
         <property name="text">
          <string>0%</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="lab_speed">
         <property name="text">
          <string>速度:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_speed_num">
         <property name="text">
          <string>0 k/s</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_file_size">
         <property name="text">
          <string>文件大小:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_file_size_num">
         <property name="text">
          <string>0 M</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>528</x>
      <y>12</y>
      <width>68</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_download">
       <property name="text">
        <string>下载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_path">
       <property name="text">
        <string>路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_stop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
